bin/cogeqc-summarize-ogs.R

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env Rscript
+library(parallel)
+
+# Be sure to set the library path to a temporary directory first. 
+dir.create('./Rlibs', showWarnings = FALSE)
+.libPaths('./Rlibs')
+BiocManager::install('cogeqc', lib = './Rlibs')
```

Comment:
I would probably have this already included in the docker image that you provide for the module that uses this script. I think there's a lot that could go wrong with calling an install as part of the script in the workflow and not having it stable inside the docker image

Reply:
Yeah, i need to make a docker image for this. I wrote this module before I knew how to do so. On the docket!